src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -596,6 +589,23 @@ public Snapshot makeSnapshotFromSnapshotRequest(SnapshotRequestModel snapshotReq
         .tags(TagUtils.sanitizeTags(snapshotRequestModel.getTags()));
   }
 
+  private void getTablesByQueryAsset(
```

Comment:
Is this name correct? It looks like it's getting the tables from the dataset, not from the query asset. In a future PR there will be two different versions, one that generates the asset from the dataset, and another that generates it from the query.

Reply:
Hm I'm not sure - it calls conjureSnapshotTablesFromAsset which made me want to call it sometyhing like getTablesByAsset... maybe getTablesFromDatasetByAsset?

Reply:
I think it's more like `createSnapshotFromDatasetAsset()`. But we have followup work to change how assets are created for requests, and this will probably change again then, so renaming it could wait until then.